django_dramatiq/management/commands/rundramatiq.py

```diff
@@ -152,31 +154,47 @@ def is_ignored_module(module_name):
             return False
 
         for conf in app_configs:
-            module = conf.name + ".tasks"
-
-            if is_ignored_module(module):
-                self.stdout.write(" * Ignored tasks module: %r" % module)
-                continue
-
-            imported_module = importlib.import_module(module)
-            if not self._is_package(imported_module):
-                self.stdout.write(" * Discovered tasks module: %r" % module)
-                tasks_modules.append(module)
-            else:
-                submodules = self._get_submodules(imported_module)
-
-                for submodule in submodules:
-                    if is_ignored_module(submodule):
-                        self.stdout.write(" * Ignored tasks module: %r" % submodule)
-                    else:
-                        self.stdout.write(" * Discovered tasks module: %r" % submodule)
-                        tasks_modules.append(submodule)
+            for task_module in DRAMATIQ_AUTODISCOVER_MODULES:
+                if conf.name == "django_dramatiq":
+                    # we want to find our own tasks, even if the user has overridden
+                    # the tasks module name
+                    module = "django_dramatiq.tasks"
+                else:
+                    module = conf.name + "." + task_module
+
+                if is_ignored_module(module):
+                    self.stdout.write(" * Ignored tasks module: %r" % module)
+                    continue
+
+                try:
+                    imported_module = importlib.import_module(module)
+                except ImportError:
+                    self.stdout.write(" * Could not import %s from %s" % (
+                        task_module, module
+                    ))
+                    continue
+                if not self._is_package(imported_module):
+                    self.stdout.write(" * Discovered tasks module: %r" % module)
+                    tasks_modules.append(module)
+                else:
+                    submodules = self._get_submodules(imported_module)
+
+                    for submodule in submodules:
+                        if is_ignored_module(submodule):
+                            self.stdout.write(" * Ignored tasks module: %r" % submodule)
+                        else:
+                            self.stdout.write(" * Discovered tasks module: %r" % submodule)
+                            tasks_modules.append(submodule)
 
         return tasks_modules
 
     def _is_package(self, module):
         module_path = getattr(module, "__path__", None)
-        return module_path and os.path.isdir(module_path[0])
+        try:
+            return module_path and os.path.isdir(module_path[0])
```

Comment:
I think it's better if this keeps failing rather than making it swallow all exceptions. It may be safe to rewrite this entire method to just

```python
def _is_package(...):
  return hasattr(module, "__path__")
```

quoting the docs:

> By definition, if a module has a `__path__` attribute, it is a package.